frontend/homework/12/App.tsx

```diff
@@ -0,0 +1,13 @@
+
+import './App.css';
+import Todolist from './Todolist';
+
+function App() {
+  return (
+    <div className="App">
+      <Todolist/>
+    </div>
```

Comment:
lets have a MainContainer here. Keep the App.tsx clean as we need to add multiple things here for a Enterprise applicaiton